sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/valueentity/ValueEntityAdapter.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk.impl.valueentity
+
+import java.util.Optional
+
+import scala.jdk.OptionConverters._
+import scala.compat.java8.DurationConverters._
+import com.akkaserverless.javasdk
+import com.akkaserverless.scalasdk.Metadata
+import com.akkaserverless.scalasdk.PassivationStrategy
+import com.akkaserverless.scalasdk.impl.MetadataImpl
+import com.akkaserverless.scalasdk.impl.valueentity.ValueEntityEffectImpl
+import com.akkaserverless.scalasdk.valueentity.CommandContext
+import com.akkaserverless.scalasdk.valueentity.ValueEntity
+import com.akkaserverless.scalasdk.valueentity.ValueEntityContext
+import com.akkaserverless.scalasdk.valueentity.ValueEntityOptions
+import com.akkaserverless.scalasdk.valueentity.ValueEntityProvider
+import com.google.protobuf.Descriptors
+
+private[scalasdk] class Scala2JavaValueEntityAdapter[S](scalasdkValueEntity: ValueEntity[S])
```

Comment:
On #433, I have used Scala2Java and Java2Scala for the Action adapter, but also introduced `JavaServiceCallAdapter`. The language prefix indicating that it adapts to Java. 

I think we can remove `Scala2` and `Java2`. Not on that PR, but nicer if we align later.

Reply:
yes that is a little easier to read

Reply:
> The language prefix indicating that it adapts to ...

yes, that would be enough